packages/app/components/Header/Header.tsx

```diff
@@ -0,0 +1,47 @@
+"use client";
+import Image from "next/image";
+import Logo from "../../public/assets/images/logo.svg";
+import Stackly from "../../public/assets/images/stackly.svg";
+import Blocks from "../../public/assets/images/blocks.svg";
+
+import Link from "next/link";
+import MobileMenu from "./MobileMenu";
+
+export default function Header() {
+  return (
+    <header className="h-[72px] sticky top-0 z-50 flex flex-col w-full bg-surface-25 border-b border-solid border-b-surface-75">
+      <nav className="h-full flex items-center w-full max-w-screen-xl px-4 mx-auto">
+        <div>
+          <Link href="/" title="Stackly Home" className="flex items-center">
+            <Image aria-label="Stackly logo" src={Logo} alt="Stackly Logo" />
+            <Image
+              aria-label="Stackly"
+              src={Stackly}
+              alt="Stackly Logo"
+              className="ml-[12px] hidden md:block"
```

Comment:
Let's use the default scale when possible. 

`ml-3 == 12px`

```suggestion
              className="ml-3 hidden md:block"
```

Reply:
This is extension makes using tailwind really easy. You can even hover on classes to get more info like the equivalent px.

**Tailwind IntelliSense**
https://github.com/tailwindlabs/tailwindcss-intellisense

**Headwind**
Keeps the classnames sorted the same.
https://github.com/heybourn/headwind